src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -177,6 +186,44 @@ void scrollY(double deltaY) {
         orientation.scrollVertically(this, deltaY);
     }
 
+    void scrollXToPixel(double pixel) {
+        orientation.scrollHorizontallyToPixel(this, pixel);
+    }
+
+    void scrollYToPixel(double pixel) {
+        orientation.scrollVerticallyToPixel(this, pixel);
+    }
+
+    Val<Double> totalWidthEstimateProperty() {
+        return Val.wrap(orientation.widthEstimateProperty(this));
```

Comment:
I.e. change [these](https://github.com/TomasMikula/Flowless/blob/master/src/main/java/org/fxmisc/flowless/OrientationHelper.java#L55-L56) to return `Val<Double>`.